### PR TITLE
Feature/Routing

### PR DESCRIPTION
**Pull request by divanc**

### CODE REVIEW

File: src/utils/buildLayout.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export function buildLayout(parent,cols) {
+  var COL_COUNT = cols; 
+  var col_heights = []
```
Comment (fletcherist) [inline src/utils/buildLayout.js:3]: use camel case for var names pls

File: src/Сomponents/Footer/Footer.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
\ No newline at end of file
+export const Footer = () => (
+  <footer className={styles.footer}>
+    <Link to='/about'>About</Link>
+    
+    <Link  to='/beginner'>What's Cut Up?</Link>
+    
+    <a href="https://github.com/Betra/duchamp-it-up"
+       target="_blank" 
+       rel='noopener noreferrer'>
```
Comment (fletcherist) [inline src/Сomponents/Footer/Footer.js:14]: not bad!!!!!!!!!
obama.jpg

File: src/Сomponents/Form/Form.js [hunk 3]
```diff
@@ -26,10 +29,10 @@ export class Form extends React.Component {
   
   render() {
     return (
-      <div>
+      <section className="formInput">
```
Comment (fletcherist) [inline src/Сomponents/Form/Form.js:32]: dont know, but it seems that 
`import './Form.module.css';`

imports css into global namespace. here you loose all the advantages of css modules

heres a better way to do it
`import css from './Fork.module.css'`
`<section className={css.formInput}>`


but actually i dont know. need some researching

**Review by fletcherist — CHANGES_REQUESTED** [review]

File: src/Сomponents/Form/Form.js (hunk 3 above)
Comment (divanc) [reply src/Сomponents/Form/Form.js:32]: Global fix: 3e5f5a7
Почему-то юзал import **{** styles **}**, из-за это часто возникали TypeError, сейчас понял

File: src/utils/buildLayout.js (hunk 1 above)
Comment (divanc) [reply src/utils/buildLayout.js:3]: Done
ca1ac1d

**Pull request merged**